src/main/java/com/nike/cerberus/cli/EnvironmentConfigToArgsMapper.java

```diff
@@ -0,0 +1,370 @@
+package com.nike.cerberus.cli;

```

Comment:
- [ ] Need to double check that we are not adding the key into args on optional parameters when the value is not present in the yaml.